ballerina/tests/basic_client_test.bal

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/mime;
+
+@test:Config {}
+function testSendReceive11() returns error? {
+    Client soapClient = check new("http://ws.cdyne.com/phoneverify/phoneverify.asmx?wsdl");
+
+    xml body = xml `<quer:CheckPhoneNumber xmlns:quer="http://ws.cdyne.com/PhoneVerify/query">
+         <quer:PhoneNumber>18006785432</quer:PhoneNumber>
+         <quer:LicenseKey>0</quer:LicenseKey>
+      </quer:CheckPhoneNumber>`;
+
+    xml|mime:Entity[]|error response = check soapClient->sendReceive(body);
```

Comment:
```suggestion
    xml|mime:Entity[] response = check soapClient->sendReceive(body);
```

check for other places too.

Reply:
Fixed in [24188b1](https://github.com/ballerina-platform/module-ballerina-soap/pull/82/commits/24188b1c4975d753810edc6d4f5af71eff23fb03)